### ISSUE TITLE
As a web client developer, I want to be able to access the API without having to ask for authorization

### ISSUE THREAD

**Issue by tloubrieu-jpl** [NASA-PDS/registry-api#192]
## 🧑‍🔬 User Persona(s)

Web developers (external and internal to JPL)

## 💪 Motivation
...so that I can start a UI development without having to ask to a specific authorization.

## 📖 Additional Details

Update the CORS parameters on the production services (`Access-Control-Allow-Origin: ...`)
- Add '*.launchbox.run' to the allowed domain or 'localhost.launchbox.run' so that external developers can use the API without a need to ask for specific authorization.
- Add `*.jpl.nasa.gov` to the allowed domain so that internal JPL developers can use the API by using localhost.jpl.nasa.gov to call their local API
- Update the documentation of the API (https://nasa-pds.github.io/pds-api/guides/search/tutorials.html#web-search-interface-tutorial-html-javascript) with how to use the API from a local web application by calling it from http://localhost.launchbox.run

## ⚖️ Acceptance Criteria
**Given** 
**When I perform** 
**Then I expect**

**Comment by jordanpadams** [comment]
@tloubrieu-jpl how critical is this? do we really need to figure this out this build cycle? I am thinking this may be a `should-have` we can punt until later when folks start trying to create web clients to access our site. for now, we can just add CORS support for specific IP ranges.

thoughts?

**Comment by tloubrieu-jpl** [comment]
Yes @jordanpadams , there's no hurry, the urgent thing was to have Eddy able to develop easily on top of the API which is done. I created this ticket to track the rest which we know need to be done for external developers.
